docs/index.md

```diff
@@ -52,19 +52,74 @@ Use Office Scripts in Excel to automate your common tasks. Explore the following
                 <div class="card">
                     <div class="cardImageOuter">
                         <div class="cardImage">
-                            <a href="develop/scripting-fundamentals.md" target="_blank"><img src="images/index-landing-page/i_code-blocks.svg" alt="Scripting fundamentals" /></a>
+                            <a href="develop/power-automate-integration.md" target="_blank"><img src="images/index-landing-page/i_architecture.svg" alt="Integrate with Power Automate"/></a>
```

Comment:
Suggestion: Add a description to be consistent with the other cards in the same section.